docs/Migration-from-the-AuthComponent.md

```diff
@@ -151,3 +151,36 @@ $this->Authentication->allowUnauthenticated(['view']);
 ```
 
 Each call to `allowUnauthenticated()` will overwrite the current action list.
+
+# Migrating Unauthorized Redirects
```

Comment:
I am still in favor of using the correct high level word "Unauthenticated" everywhere.
It just confuses me and probably others as at this point - especially since we do that distinction even with the two separate plugins - there is not even authentication happened yet. And everyone learns Authorization comes after successful Authentication.

Reply:
Changed the name of the middleware to `UnauthenticatedRedirectMiddleware`. I'll rename the exception separately.